app/controllers/email_address_controller.rb

```diff
@@ -0,0 +1,24 @@
+class EmailAddressController < ApplicationController
+	def destroy
+		render json: {
+			:status => EmailAddress.delete(params[:id])
+		}
+	end
+	def create
+		begin
+			status = EmailAddress.new(email_address_params).save!
```

Comment:
Instead of begin rescue, I could use ```save``` and not ```save!``` 
Also by using save it will return you a set of errors on the object.